cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/prereqs.yaml

```diff
@@ -1,21 +1,19 @@
 
 prereqs:
   prereqDefs:
-    ibmCoreServices: {}
-    k8sDistros: {}
-      #ibmCloud:
-      #  distribution: ibmCloud
-      #  semver: '>=1.11.3'
-      #ibmCloudPrivate:
-      #  distribution: ibmCloudPrivate
-      #  semver: '>=1.11.3'
-      #kubernetes:
-      #  distribution: kubernetes
-      #  semver: '>=1.14'
-      #openshift:
-      #  distribution: openshift
-      #  semver: '>=4.2'
-    k8sResources:  {}
+    k8sDistros: 
```

Comment:
We had the discussion that the prerequisite definitions here mean you need to use them and when you don't, produce this error:
```
[ERROR]: The prereq item kubernetes, is declared but not used in any actions for any inventory item (CASEProcessingError)
[ERROR]: The prereq item openshift, is declared but not used in any actions for any inventory item (CASEProcessingError)
```

If you guys will not be supporting CASE based installation, I would revert the changes made to this file so that the `casectl verify` command could execute successfully.